.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
yes sorry about that, i thought it was in .gitignore
the .idea directory and the *.sh files should not be there